<!doctype html>
<!-- 
##############################################################
Ignore this boilerplate if you're just trying to edit the text.
Skip to the part that says 'The real text begins here'
##############################################################

Based on this theme: https://github.com/broccolini/dinky , which mentioned that attribution is appreciated. Thanks, broccolini!
-->
<html lang="en">
  <head>
    <base target="_blank">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="citation_title" content="Split-wrmScarlet and split-sfGFP: tools for faster, easier fluorescent labeling of endogenous proteins in Caenorhabditis elegans">
    <meta name="citation_author" content="Je&#769;ro&#770;me Goudeau">
    <meta name="citation_author" content="Jonathan Paw">
    <meta name="citation_author" content="Laura Savy">
    <meta name="citation_author" content="Manuel D. Leonetti">
    <meta name="citation_author" content="Andrew G. York">
    <meta name="citation_author" content="Cynthia Kenyon">
    <meta name="citation_author" content="Maria Ingaramo">
    <meta name="citation_publication_date" content="2020/07/03">
    <meta name="citation_journal_title" content="Github.io">
    <meta name="citation_pdf_url" content="https://andrewgyork.github.io/split_wrmscarlet.pdf">
    <title>Split Fluorescent Proteins for C. elegans</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/prism.css">
    <!--[if lt IE 9]>
    <script src="javascript/html5shiv/html5shiv.js"></script>
    <![endif]-->
    <!--[if lt IE 8]>
      <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
    <![endif]-->
    <script src="javascript/scale-fix/scale.fix.js"></script>
    <script src="javascript/python-highlighting/prism.js"></script>
    <script async  src="javascript/Minimal-MathJax/MathJax.js?config=TeX-AMS_CHTML"></script>
    <script src="javascript/update_figures.js"></script>
    <script src="javascript/reference_list/reference_list.js"></script>
    <script src="javascript/google-analytics/analytics.js"></script>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
  </head>
  <body>

	  
    <div class="wrapper">
      <header>
        <h1 class="header">Split Fluorescent Proteins for <em>C. elegans</em></h1>
        <ul>
          <li class="download"><a class="buttons" href="https://github.com/AndrewGYork/split_wrmscarlet/zipball/master">Download ZIP</a></li>
          <li><a class="buttons github" href="https://github.com/AndrewGYork/split_wrmscarlet">View On GitHub</a></li>
          <li><a class="buttons pdf" href="./split_wrmscarlet.pdf">Download PDF</a></li>
        </ul>
        
        <p class="header">This project is maintained by <a class="header name" href="https://github.com/marimar128">Maria Ingaramo</a> in the <a class="header name" href="http://andrewgyork.github.io/">York lab</a>, and was funded by <a class="header name" href="https://www.calicolabs.com/">Calico Life Sciences LLC</a></p>
      </header>

	  
	  
<!-- 
##############################################################
The real text begins here.
##############################################################
 -->
<section>
<h1>Research Article</h1>

<h3 class="onlyprint">
Note that this is a limited PDF or print version; animated and interactive figures are disabled. For the full version of this article, please visit one of the following links:
<a href="https://andrewgyork.github.io/split_wrmscarlet">https://andrewgyork.github.io/split_wrmscarlet</a> <a href="https://calico.github.io/split_wrmscarlet">https://calico.github.io/split_wrmscarlet</a></h3>
<noscript>
<h3>Your browser doesn't seem to support Javascript. This document uses Javascript for interactive figures, math typesetting, and to automatically generate the reference list. Either activate Javascript, or use the "Download PDF" link above if you want properly typeset math and a reference list.</h3></noscript>

<h2>Split-wrmScarlet and split-sfGFP: tools for faster, easier fluorescent labeling of endogenous proteins in <em>Caenorhabditis elegans</em></h2>

<p class="author_list">
Je&#769;ro&#770;me Goudeau<sup>1,3</sup>, Jonathan Paw<sup>1</sup>, Laura Savy<sup>2</sup>, Manuel D. Leonetti<sup>2</sup>, Andrew G. York<sup>1</sup>, Cynthia Kenyon<sup>1,3</sup> and Maria Ingaramo<sup>1,3</sup></p>

<p class="author_affiliations"><sup>1</sup>Calico Life Sciences LLC, South San Francisco, CA, USA</p>
<p class="author_affiliations"><sup>2</sup>Chan Zuckerberg Biohub, San Francisco, California 94158, United States</p>

<p class="contact_email"><sup>3</sup>Corresponding authors: 
<a href="mailto:jerome@calicolabs.com" target="_self">jerome@calicolabs.com</a>, 
<a href="mailto:cynthia@calicolabs.com" target="_self">cynthia@calicolabs.com</a>, 
<a href="mailto:ingaramo@calicolabs.com" target="_self">ingaramo@calicolabs.com</a></p>


<a id="Abstract" href="#Abstract" target="_self">
<h4>Abstract</h4></a>
<p class="abstract">We create and share a new red fluorophore, along with a set of strains, reagents and protocols, to make it faster and easier to label endogenous <em>C. elegans</em> proteins with fluorescent tags. CRISPR-mediated fluorescent labeling of <em>C. elegans</em> proteins is an invaluable tool, but it is much more difficult to insert fluorophore-size DNA segments than it is to make small gene edits. In principle, high-affinity asymmetrically split fluorescent proteins solve this problem in <em>C. elegans</em>: the small fragment can quickly and easily be fused to almost any protein of interest and can be detected wherever the large fragment is expressed and complemented. There is currently only one available strain stably expressing the large fragment of a split fluorescent protein, restricting this solution to a single tissue (the germline) in the highly autofluorescent green channel. No available <em>C. elegans</em> lines express unbound large fragments of split red fluorescent proteins, and even state-of-the-art split red fluorescent proteins are dim compared to the canonical split-sfGFP protein. In this study, we engineer a bright, high-affinity new split red fluorophore, <a href="https://www.addgene.org/138966/">split-wrmScarlet</a>, and generate transgenic <em>C. elegans</em> lines to allow easy <a href="https://cgc.umn.edu/strain/CF4582">single-color</a> labeling in <a href="https://cgc.umn.edu/strain/CF4610">muscles</a> and <a href="https://cgc.umn.edu/strain/CF4588">dual-color</a> labeling in somatic cells. We validate these strains by targeting split-wrmScarlet to several genes whose products label distinct organelles, and we provide a <a href="https://doi.org/10.17504/protocols.io.bamkic4w">protocol</a> for an easy, cloning-free method for CRISPR/Cas9 editing.
</p>

<h3>Peer review status</h3>
<p class="abstract">Pre-print published July 03, 2020 (This article is not yet peer-reviewed)</p>
<p class="abstract">Cite as: <a href="https://doi.org/10.5281/zenodo.xxxxxxx">doi:10.5281/zenodo.xxxxxxx</a></p>

<a id="Introduction" href="#Introduction" target="_self">
<h3>Introduction</h3></a>

<p>Genetically-expressed fluorophores are essential tools for visualizing and quantifying cellular proteins. In <em>C. elegans</em>, fluorescent proteins have traditionally been introduced on extrachromosomal arrays 

[<a class="citation" href="https://doi.org/10.1038/299456a0" title="Kimble, J., Hodgkin, J., Smith, T. &amp; Smith, J. &quot;Suppression of an amber mutation by microinjection of suppressor tRNA in C. elegans.&quot; Nature 299, 456–458 (1982).">Kimble 1982</a>, 
<a class="citation" href="https://doi.org/10.1002/j.1460-2075.1991.tb04966.x " title="Mello, C. C., Kramer, J. M., Stinchcomb, D. &amp; Ambros, V. &quot;Efficient gene transfer in C. elegans: extrachromosomal maintenance and integration of transforming sequences.&quot; The EMBO Journal 10, 3959–3970 (1991). ">Mello 1991</a>]

 or via MosSCI-based integration 
 
 [<a class="citation" href="https://doi.org/10.1038/nmeth.1865" title="Frøkjær-Jensen, C., Davis, M. W., Ailion, M. &amp; Jorgensen, E. M.  &quot;Improved Mos1-mediated transgenesis in C. elegans.&quot; Nature Publishing Group 9, 117–118 (2012).">Frøkjær-Jensen 2012</a>, 
 <a class="citation" href="https://doi.org/10.1038/ng.248" title="Frøkjær-Jensen, C. et al. &quot;Single-copy insertion of transgenes in Caenorhabditis elegans.&quot; Nat Genet 40, 1375–1383 (2008).">Frøkjær-Jensen, C. 2008</a>]. 
 
 These methods have enabled important discoveries but can also lead to artifacts due to supraphysiological gene-expression levels and lack of endogenous regulatory control. In recent years, the repertoire of <em>C. elegans</em> transgenic tools has expanded 
 
 [see <a class="citation" href="https://doi.org/10.1534/genetics.119.301506" title="Nance, J. &amp; Frøkjær-Jensen, C. &quot;The Caenorhabditis elegans Transgenic Toolbox.&quot; The EMBO Journal 10, 3959–3970 (1991).">Nance 2019</a> for review], 
 
 particularly due to advances in CRISPR/Cas9 genome-editing technologies 
 
 [<a class="citation" href="https://doi.org/10.1534/genetics.114.170423" title="Paix, A. et al. &quot;Scalable and versatile genome editing using linear DNAs with microhomology to Cas9 Sites in Caenorhabditis elegans.&quot; Genetics 198, 1347–1356(2014).">Paix 2014</a>, 
 <a class="citation" href="https://doi.org/10.1534/genetics.115.182162" title="Dickinson, D. J. &amp; Goldstein, B. &quot;CRISPR-Based Methods for Caenorhabditis elegans Genome Engineering.&quot; Genetics 202, 885–901 (2016).">Dickinson 2016</a>]. 
 
 CRISPR/Cas9 allows precise transgene insertion by homology-directed repair (HDR) and can be used to label an endogenous gene at its native locus with a fluorescent protein 
 
 [<a class="citation" href="https://doi.org/10.1534/genetics.118.301532" title="Dokshin, G. A., Ghanta, K. S., Piscopo, K. M. &amp; Mello, C. C.  &quot;Robust Genome Editing with Short Single-Stranded and Long, Partially Single-Stranded DNA Donors in Caenorhabditis elegans.&quot; Genetics 210, 781–787 (2018).">Dokshin 2018</a>, 
 <a class="citation" href="https://doi.org/10.1534/genetics.118.301775" title="Farboud, B., Severson, A. F. &amp; Meyer, B. J. &quot;Strategies for Efficient Genome Editing Using CRISPR-Cas9.&quot; Genetics 211, 431–457 (2019).">Farboud 2019</a>,
  <a class="citation" href="https://doi.org/10.1534/genetics.119.301965" title="Vicencio, J., Martínez-Fernández, C., Serrat, X. &amp; Cerón, J. &quot;Efficient Generation of Endogenous Fluorescent Reporters by Nested CRISPR in Caenorhabditis elegans.&quot; Genetics 211, 1143–1154 (2019).">Vicencio 2019</a>].
 
 </p>
 
<p>However, relative to CRISPR/Cas9-mediated integration of smaller transgenes, genomic insertion of large DNA fragments like those encoding fluorescent proteins remains a challenge, both because repair with double-stranded templates is less efficient than repair with single-stranded oligodeoxynucleotide donors (ssODN) 

[<a class="citation" href="https://doi.org/10.1534/genetics.118.301775" title="Farboud, B., Severson, A. F. &amp; Meyer, B. J. &quot;Strategies for Efficient Genome Editing Using CRISPR-Cas9.&quot; Genetics 211, 431–457 (2019).">Farboud 2019</a>]

, and because of the requirement for cloning to prepare the HDR donor template. Recent methods such as ‘hybrid’ 

[<a class="citation" href="https://doi.org/10.1534/genetics.118.301532" title="Dokshin, G. A., Ghanta, K. S., Piscopo, K. M. &amp; Mello, C. C. &quot;Robust Genome Editing with Short Single-Stranded and Long, Partially Single-Stranded DNA Donors in Caenorhabditis elegans.&quot; Genetics 210, 781–787 (2018).">Dokshin 2018</a>]

 and ‘nested’ 
 
 [<a class="citation" href="https://doi.org/10.1534/genetics.119.301965" title="Vicencio, J., Martínez-Fernández, C., Serrat, X. &amp; Cerón, J. &quot;Efficient Generation of Endogenous Fluorescent Reporters by Nested CRISPR in Caenorhabditis elegans.&quot; Genetics 211, 1143–1154 (2019).">Vicencio 2019</a>]
 
  CRISPR remove the need for cloning but still require preparation of the DNA template or several rounds of injections and selection of transgenic progeny. As a result, using CRISPR with small ssODN templates is currently faster, easier, cheaper and more efficient than with large templates. In our lab, we routinely make <em>C. elegans</em> genome edits with short ssODN with almost guaranteed success. In contrast, in our experience, large edits using double-stranded DNA templates are more time-consuming and have higher failure rates.</p>
  
  
<p>Our preferred approach is to combine the utility of full-length fluorescent proteins with the convenience of short genomic edits, by using high-affinity asymmetrically-split fluorescent proteins 

[<a class="citation" href="https://doi.org/10.1038/nbt1044" title="Cabantous, S., Terwilliger, T. C. &amp; Waldo, G. S. &quot;Protein tagging and detection with engineered self-assembling fragments of green fluorescent protein.&quot; Nat Biotechnol 23, 102–107 (2004).">Cabantous 2004</a>]

. These fluorophores typically separate a GFP-like protein between the 10th and 11th strands of the beta barrel, splitting it asymmetrically into a large (FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>) and a small (FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>) fragment. The fragments are not individually fluorescent, but upon binding one another, recapitulate the fluorescent properties of an intact fluorophore (Figure 1A). Unlike the low-affinity split fluorescent proteins used in BiFC assays 

[<a class="citation" href="https://doi.org/10.1016/S1097-2765(02)00496-3" title="Hu, C. D., Chinenov, Y. &amp; Kerppola TK. &quot;Visualization of interactions among bZIP and Rel family proteins in living cells using bimolecular fluorescence complementation.&quot; Molecular Cell 9, 789-98 (2002). ">Hu 2002</a>]

, high-affinity binding between the fragments is critical here. Our preferred approach for tagging a new cellular protein begins with a <em>C. elegans</em> strain expressing the large FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> fragment in cells of interest, unattached to any cellular protein. This way, only the small FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> fragment (&lt;60 nt) needs to be inserted to tag the target protein, which will only fluoresce in compartments where it can bind the large fragment. These short insertions tend to be faster, easier, and more reliable to make than inserting a >600 nt full-length fluorescent protein 

[<a class="citation" href="https://doi.org/10.1534/genetics.115.179382" title="Paix, A. et al. &quot;High Efficiency, Homology-Directed Genome Editing in Caenorhabditis elegans Using CRISPR-Cas9 Ribonucleoprotein Complexes.&quot; Genetics 201, 47-54 (2015).">Paix 2015</a>, 
<a class="citation" href="https://doi.org/10.1534/g3.117.300216" title="Prior, H., Jawad, A. K., MacConnachie, L. &amp; Beg, A. A. &quot;Highly Efficient, Rapid and Co-CRISPR-Independent Genome Editing in Caenorhabditis elegans.&quot; G3 7, 3693–3698 (2017).">Prior 2017</a>,
 <a class="citation" href="https://doi.org/10.1534/genetics.118.301532" title="Dokshin, G. A., Ghanta, K. S., Piscopo, K. M. &amp; Mello, C. C. &quot;Robust Genome Editing with Short Single-Stranded and Long, Partially Single-Stranded DNA Donors in Caenorhabditis elegans.&quot; Genetics 210, 781–787 (2018).">Dokshin 2018</a>,
  <a class="citation" href="https://doi.org/10.1038/s41588-018-0174-0" title="Richardson, C. D. et al. &quot;CRISPR-Cas9 genome editing in human cells occurs via the Fanconi anemia pathway.&quot; Nat Genet 50, 1132–1139 (2018).">Richardson 2018</a>]

. Therefore, collections of <em>C. elegans</em> lines stably expressing the large FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in different tissues are an invaluable resource allowing rapid fluorescent tagging in a cell type of choice. Stable lines with red FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> fragments would be especially useful, given <em>C. elegans</em>’ substantial autofluorescence in the GFP channel.</p>
<p>Green and red asymmetrically-split fluorescent proteins have been used to combine cell and protein specificity in <em>C. elegans</em> neurons and synapses 

[<a class="citation" href="https://doi.org/10.7554/eLife.26376" title="Noma, K., Goncharov, A., Ellisman, M. H. &amp; Jin, Y. &quot;Microtubule-dependent ribosome localization in C. elegans neurons.&quot; eLife 6, 218 (2017).">Noma 2017</a>, 
<a class="citation" href="https://doi.org/10.1534/genetics.119.302063" title="He, S., Cuentas-Condori, A. &amp; Miller, D. M. &quot;NATF (Native and Tissue-Specific Fluorescence): A Strategy for Bright, Tissue-Specific GFP Labeling of Native Proteins in Caenorhabditis elegans.&quot; Genetics 212, 387–395 (2019).">He 2019</a>, 
<a class="citation" href="https://doi.org/10.1038/s42003-019-0589-x" title="Feng, S. et al. &quot;Bright split red fluorescent proteins for the visualization of endogenous proteins and synapses.&quot; Communications Biology 2, 344–12 (2019).">Feng 2019</a>]

; however, these strains used extrachromosomal arrays, not stable lines, which are more time-consuming to maintain and can have variable expression levels. To the best of our knowledge, there is only one available unbound FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> stable <em>C. elegans</em> line, which expresses sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in the germline 

[<a class="citation" href="https://doi.org/10.1534/g3.119.400162" title="Hefel, A. &amp; Smolikove, S. &quot;Tissue-Specific Split sfGFP System for Streamlined Expression of GFP Tagged Proteins in the Caenorhabditis elegans Germline.&quot; G3 9, 1933–1943 (2019). ">Hefel 2019</a>]

, and there are no available lines with red FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> fragments. Existing red split fluorophores are also much dimmer in <em>C. elegans</em> than green ones, despite recent improvements like split-sfCherry3 

[<a class="citation" href="https://doi.org/10.1038/s42003-019-0589-x" title="Feng, S. et al. &quot;Bright split red fluorescent proteins for the visualization of endogenous proteins and synapses.&quot; Communications Biology 2, 344–12 (2019).">Feng 2019</a>]

.</p>
<p>Here, we describe tools that reduce these obstacles for convenient fluorescent labeling of endogenous <em>C. elegans</em> proteins. We engineered a new split red fluorescent protein based on mScarlet 

[<a class="citation" href="https://doi.org/10.1038/nmeth.4074" title="Bindels, D.S. et al. &quot;mScarlet: a bright monomeric red fluorescent protein for cellular imaging.&quot; Nature Publishing Group 14, 53–56 2016.">Bindels 2016</a>, 
<a class="citation" href="https://doi.org/10.1534/g3.117.040824" title="El Mouridi, S. et al. &quot;Reliable CRISPR/Cas9 Genome Engineering in Caenorhabditis elegans Using a Single Efficient sgRNA and an Easily Recognizable Phenotype.&quot; G3 7, 1429–1437 (2017).">El Mouridi 2017</a>]

, which is three times brighter in worms than split-sfCherry3 (<a href="https://www.addgene.org/138966/">https://www.addgene.org/138966/</a>). We generated <em>C. elegans</em> lines carrying single-copy insertions of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> expressed in somatic cells (<a href="https://cgc.umn.edu/strain/CF4582">https://cgc.umn.edu/strain/CF4582</a>) and in muscle (<a href="https://cgc.umn.edu/strain/CF4610">https://cgc.umn.edu/strain/CF4610</a>), and are making them available to the <em>C. elegans</em> community. We provide a protocol for an easy, cloning-free method to label endogenous genes with FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>s using CRISPR/Cas9, commercially available synthetic single-stranded oligodeoxynucleotide (ssODN) donors, and microinjection (<a href="https://doi.org/10.17504/protocols.io.bamkic4w">doi.org/10.17504/protocols.io.bamkic4w</a>). We validate this protocol by targeting wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> to six different genes whose products have distinct cellular locations. We also show that labeling with tandem wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>-repeats increases fluorescence in vivo, and we provide the plasmid necessary to generate the dsDNA template through Addgene (link pending, expected in early August 2020). We also generated a strain expressing an integrated copy of sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> 

[<a class="citation" href="https://doi.org/10.1038/nbt1172" title="Pédelacq, J.-D., Cabantous, S., Tran, T., Terwilliger &amp; T. C., Waldo, G. S. &quot;Engineering and characterization of a superfolder green fluorescent protein.&quot; Nature Biotechnology 24(1), 79-88 (2005)">Pédelacq 2005</a>] 

in somatic cells (<a href="https://cgc.umn.edu/strain/CF4587">https://cgc.umn.edu/strain/CF4587</a>). Finally, to further expand the toolkit, we generated a dual-color strain expressing both sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> and wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in somatic cells (<a href="https://cgc.umn.edu/strain/CF4588">https://cgc.umn.edu/strain/CF4588</a>), for two-color applications such as colocalization studies or organelle interaction. We hope that these resources will facilitate the study of <em>C. elegans</em> biology.</p>

<a id="Methods" href="#Methods" target="_self">
<h3>Materials and Methods</h3></a>
<p><strong>Mutagenesis and screening. </strong>
For the initial screenings in <em>E. coli</em>, we introduced a 32 amino-acid spacer between the tenth and eleventh β-strands of full-length wrmScarlet in a pRSET vector 

[<a class="citation" href="https://doi.org/10.1038/s41467-017-00494-8" title="Feng, S. et al. &quot;Improved split fluorescent proteins for endogenous protein labeling.&quot; Nature Communications 8, 370 (2017).">Feng 2017</a>]

. This starting construct was non-fluorescent, but we restored low fluorescence levels by introducing the superfolder mutation G220A. Semi-random mutagenesis was carried out using rolling-circle amplification with NNK primers at positions I8, K10, F15, G32, Q43, A45, K46, L47, G52, G53, D60, S63, P64, Q65, F66, S70, R71, T74, K75, D79, Y84, W94, R96, T107, V108, Q110, E115, L125, R126, T128, K139, K140, W144, E145, S147, T148, E149, R150, I162, K163, M164, L175, F178, K179, K183, K185, K186, N195, R198, I202, T203, S204, D208, Y209, T210, V211, V212, E213, Q214, Y215, E216, R217, S218, E219, A220, H222, S223, T224, G225, G226, M227, D228, and E229 with Phusion polymerase (NEB) in GC buffer, followed by pooling of the PCR products, DpnI digestion and transformation into BL21(DE3) <em>E. coli</em>. These positions covered areas deemed important for brightness or stability, and the interface between FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> and FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>. Primers were resynthesized if a mutation interfered with neighboring mutagenic primer binding. The brightest three to five colonies were identified using a Leica M165 FC fluorescent stereomicroscope, and their plasmid DNA subjected to a new mutagenesis round. After five rounds, we separated the two fragments of a version of split wrmScarlet that had fluorescence comparable to the parent protein into two <em>S. cerevisiae</em> plasmids to test for complementation. Since we did not detect fluorescence, we continued selection using two plasmids in yeast. For screening on two plasmids, a pRSET vector expressing mScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> and a pD881-MR vector (ATUM) expressing mTagBFP-mScarletS<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> (without the MDELYK tail from the C-terminus) were used to perform the semi-random mutagenesis. The libraries were co-electroporated into <em>E. coli</em> and expression induced with 1% rhamnose and 1mM IPTG. The library was enriched for fluorescent clones using FACS, and then subcloned to make pRS-GPD-wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> and p416-TEF-membrane-mTagBFP-wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>. The yeast plasmids were co-transformed into a URA<sup>-</sup>, HIS<sup>-</sup>, LEU<sup>-</sup>, MET<sup>-</sup> <em>S. cerevisiae</em> strain and selected for in SC media without uracil and histidine, and FACS was used again for enrichment of clones with the highest red to blue ratio. After three rounds of semi-random mutagenesis with the two plasmids strategy, a final round of random mutagenesis was performed using the GeneMorph II kit (Agilent). Yeast plasmids will be available through Addgene in early August 2020.
</p>

<p><strong><em>C. elegans</em> strains and maintenance.</strong> Animals were cultured under standard growth conditions with <em>E. coli</em> OP50 at 20°C 

[<a class="citation" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1213120/pdf/71.pdf" title="Brenner, S. &quot; The genetics of Caenorhabditis elegans.&quot; Genetics 77, 71–94 (1974).">Brenner 1974</a>]

. Strains generated in this work are listed in the Supplementary Material, <a href="./appendix.html#Tables">Supplementary Tables.</a></p>

<p><strong>Nucleic acid reagents.</strong> Synthetic nucleic acids were purchased from Integrated DNA Technologies (IDT), GenScript or Genewiz. For knock-in of a single wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> or sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> sequence, 200-mer HDR templates were ordered in ssODN form (synthetic single-stranded oligodeoxynucleotide donors) from IDT. For knock-in of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> repeats, HDR template were ordered in dsDNA form (plasmids) from GenScript or Genewiz. For plasmids injected as extrachromosomal arrays, sequences were synthesized and cloned into the pUC57 vector (Genewiz). The complete set of crRNAs and DNA sequences used for the experiments described here can be found in Supplementary Material, <a href="./appendix.html#Tables">Supplementary Tables.</a></p>

<p><strong>Strain generation: CRISPR/Cas9-triggered homologous recombination.</strong> CRISPR insertions were performed using published protocols 

[<a class="citation" href="https://doi.org/10.1534/genetics.115.179382" title="Paix, A. et al. &quot;High Efficiency, Homology-Directed Genome Editing in Caenorhabditis elegans Using CRISPR-Cas9 Ribonucleoprotein Complexes.&quot; Genetics 201, 47-54 (2015).">Paix 2015</a>, 
<a class="citation" href="https://doi.org/10.1093/nar/gkw502" title="Paix, A., Schmidt, H. &amp; Seydoux, G. &quot;Cas9-assisted recombineering in C. elegans:genome editing using in vivo assembly of linear DNAs.&quot; Nucleic Acids Res 44, e128 (2016).">Paix 2016</a>]

. Ribonucleoprotein complexes (protein Cas9, tracrRNA, crRNA) and DNA templates were microinjected into the gonad of young adults using standard methods 

[<a class="citation" href="https://doi/10.1895/wormbook.1.108.1" title="Evans, T. C., ed. &quot;Transformation and microinjection&quot; WormBook (2016).">Evans 2016</a>]

. Injected worms were singled and placed at 25ºC overnight. All crRNA and DNA template sequences used to generate the strains described in this work are listed in the Supplementary Material, <a href="./appendix.html#Tables">Supplementary Tables</a>. WrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> and sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> integrants were identified by screening for fluorescence in the F1 or F2 progeny of injected worms. The co-CRISPR <em>dpy-10(cn64)</em> mutation was used as a marker when generating non-fluorescent strains. The CF4582 strain (<em>muIs252[Peft-3::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3'UTR, Cbr-unc-119(+)] II; unc-119(ed3) III</em>) was generated by replacing the <em>tir-1::mRuby</em> sequence from the strain CA1200 (ieSi57 II; unc-119(ed3) III 

[<a class="citation" href="https://doi.org/10.1242/dev.129635" title="Zhang, L., Ward, J. D., Cheng, Z. &amp; Dernburg, A. F. &quot;The auxin-inducible degradation (AID) system enables versatile conditional protein depletion in C.elegans.&quot; Development 142, 4374–4384 (2015).">Zhang 2015</a>]

 with the wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> sequence. The CF4587 strain (<em>muIs253[(Peft-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3'UTR, Cbr-unc-119(+)] II; unc-119(ed3) III</em>) was generated by replacing the <em>let-858</em> promoter from the strain COP1795 (<em>knuSi785 [pNU1687(Plet-858::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3’UTR, unc-119(+)] II; unc-119(ed3) III﻿</em>) with the <em>eft-3</em> promoter. Both CF4582 and CF4587 strains were generated using long, partially single-stranded DNA donors 
 
 [<a class="citation" href="https://doi.org/10.1534/genetics.118.301532" title="Dokshin, G. A., Ghanta, K. S., Piscopo, K. M. &amp; Mello, C. C. &quot;Robust Genome Editing with Short Single-Stranded and Long, Partially Single-Stranded DNA Donors in Caenorhabditis elegans.&quot; Genetics 210, 781–787 (2018).">Dokshin 2018</a>]
 
 . The CF4610 strain (<em>muIs257[Pmyo-3::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3'UTR] I</em>) was generated by inserting the wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> sequence in the WBM1126 strain following the SKI LODGE protocol 
 
 [<a class="citation" href="https://doi.org/10.1534/g3.119.400314" title="Silva-García, C. G. et al. &quot;Single-Copy Knock-In Loci for Defined Gene Expression in Caenorhabditis elegans.&quot; G3 9, 2195–2198 (2019).">Silva-García 2019</a>]
 
 . The strain PHX731 (<em>vha-13(syb731[wrmScarlet::vha-13]) V</em>) was generated by SunyBiotech using CRISPR services. Strains generated were genotyped by Sanger sequencing of purified PCR products (Genewiz).</p>

<p><strong>Strain generation: Mos1-mediated single-copy insertion.</strong> The COP1795 strain was generated by NemaMetrix using MosSCI services. The PHX1797 strain was generated by SunyBiotech using MosSCI services.</p>

<p><strong>Strain generation: genetic crosses.</strong> The following <em>C. elegans</em> strains were created by standard genetic crosses: CF4588 (<em>muIs253[Peft-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3'UTR, Cbr-unc-119(+)], muIs252[Peft-3::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3'UTR, Cbr-unc-119(+)] II; unc-119(ed3) III) and CF4602 (muIs253[Peft-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3'UTR, Cbr-unc-119(+)], muIs252[Peft-3::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3'UTR, Cbr-unc-119(+)] II; unc-119(ed3) III; fib-1(muIs254[wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::fib-1]), his-3(muIs255[his-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>]) V</em>). Non-fluorescent parental lines CF4582, CF4587 and CF4610 generated using <em>dpy-10(cn64)</em> co-CRISPR were backcrossed at least once.</p>

<p><strong>Strain generation: plasmid microinjection.</strong> <em>Peft-3::3NLS::mTagBFP2::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::T2A::mNeonGreen::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::fib-1 3’UTR, Peft-3::3NLS::mTagBFP2::sfCherry3<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::T2A::mNeonGreen::sfCherry3<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::fib-1 3’UTR, Pmyo-3::mTagBFP2::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::T2A::mNeonGreen::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::fib-1 3’UTR, or Pmyo-3::mTagBFP2::sfCherry3<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::T2A::mNeonGreen::sfCherry3<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::fib-1 3’UTR</em> constructs were microinjected at (20 ng.μL−1) using a standard microinjection procedure 

[<a class="citation" href="https://doi.org/10.1002/j.1460-2075.1991.tb04966.x " title="Mello, C. C., Kramer, J. M., Stinchcomb, D. &amp; Ambros, V. &quot;Efficient gene transfer in C. elegans: extrachromosomal maintenance and integration of transforming sequences.&quot; The EMBO Journal 10, 3959–3970 (1991). ">Mello 1991</a>]

. Germline gene expression was achieved using a microinjection-based protocol with diluted transgenic DNA 

[<a class="citation" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1207937/pdf/ge1461227.pdf" title="Kelly, W. G., Xu, S., Montgomery, M. K. &amp; Fire, A. &quot;Distinct Requirements for Somatic and Germline Expression of a Generally Expressed Caenorhabditis elegans Gene.&quot; Genetics 146, 227–238 (1997). ">Kelly 1997</a>]

, <em>Psun-1::mNeonGreen::linker::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::tbb-2 3’UTR</em> construct (5 ng/µL) was co-injected with PvuII-digested genomic DNA fragments from <em>E. coli</em> (100 ng/µL).</p>

<p><strong>Microscopy.</strong> Confocal fluorescence imaging was performed using the NIS Elements imaging software on a Nikon confocal spinning disk system equipped with an Andor EMCCD camera, a CSU-X1 confocal scanner (Yokogawa), 405, 488, and 561 nm solid-state lasers, and 455/50, 525/26 and 605/70 nm emission filters. Transgenic animals expressing sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> or wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> were screened using a Leica M165 FC fluorescent stereomicroscope equipped with a Sola SE-V with GFP and mCherry filters. </p>

<p><strong>Image analysis.</strong> Images were analyzed using Fiji. Image manipulations consisted of maximum intensity projections along the axial dimension, rolling ball radius background subtraction, smoothing, and LUT minimum and maximum adjustments. Masks were created by thresholding and setting the pixels under the threshold cutoff to NaN. Plotting of values per pixel was carried out in python 3, using numpy and matplotlib. When performing normalizations for split-sfCherry3 vs split-wrmScarlet, the red channel was divided by the green channel (mNeonGreen-FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>) because the localization of both fragments is expected to be cytosolic. For normalization of signals where mTagBFP-FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> is targeted to the membrane, the blue channel was used instead.</p>

<p><strong>Mounting worms for microscopy.</strong> Pads made of 3% agarose (GeneMate) were dried briefly on Kimwipes (Kimtech) and transferred to microscope slides. Around 10 μL of 2 mM levamisole (Sigma) was pipetted onto the center of the agarose pads. Animals were transferred to the levamisole drop, and a cover slip was placed on top before imaging.</p>

<p><strong>Brood size analysis.</strong> Eight single synchronized adults grown at 20°C were transferred to fresh plates every 24 hours until cessation of reproduction, and the number of viable progeny produced by each worm was scored.</p>

<p><strong>Developmental toxicity assay.</strong> Ten N2E wild-type animals were microinjected with either <em>Peft-3::3NLS::mTagBFP2::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::T2A::mNeonGreen::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::fib-1 3’UTR</em> or <em>Peft-3::3NLS::mTagBFP2::sfCherry3<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::T2A::mNeonGreen::sfCherry3<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::fib-1 3’UTR</em> construct at (20 ng/μL) and were singled. mNeonGreen-positive F1 animals were scored and their development was monitored for up to five days from egg-laying. The number of fluorescent dead eggs, arrested larvae (i.e. animals never reaching adulthood) or adults were scored for each group.</p>

<p><strong>Lifespan assays.</strong> NGM plates were supplemented with 5-Fluorouracil (5-FU, Sigma, 15 μM) 

[<a class="citation" href="https://doi.org/10.1371/journal.pbio.1000599" title="Goudeau, J. et al. &quot;Fatty acid desaturation links germ cell loss to longevity through NHR-80/HNF4 in C. elegans.&quot; PLoS Biol 9, e1000599.">Goudeau 2011</a>]

 in order to prevent progeny from hatching and with Kanamycin sulfate to prevent bacterial contamination (Sigma, 25 μg/mL). Animals fed with kanamycin-resistant OP50 were scored manually as dead or alive, from their L4 larval stage defined as day 0. A worm was considered alive if it moved spontaneously or, in cases where it wasn’t moving, if it responded to a light touch stimulus with a platinum wire. Animals that crawled off the plates, had eggs that accumulated internally, burrowed or ruptured were censored and included in the analysis until the time of censorship.</p>

<p><strong>Structure prediction and rendering of split-wrmScarlet.</strong> Phyre2 was used to predict the three-dimensional modelling in intensive mode with default parameters 

[<a class="citation" href="https://doi.org/10.1038/nprot.2015.053" title=" Kelley, L. A., Mezulis, S., Yates, C. M., Wass, M. N. &amp; Sternberg, M. J. E. &quot;The Phyre2 web portal for protein modeling, prediction and analysis.&quot; Nat Protoc 10, 845–858 (2015). ">Kelley 2015</a>]

. The 3D model obtained was visualized using PyMOL (v 2.2.0).</p>

<p><strong>Statistical analysis.</strong> Differences in fluorescence intensity between groups were compared using unpaired t-test with Welch’s correction. Data are presented as means ± SD. Kaplan-Meier estimates of survival curves were calculated using survival (v 2.38–3) and rms (v 4.5–0) R packages and differences were tested using log-rank test. The number of animals used in each experiment is indicated in the figure legends.</p>

<p><strong>Data availability.</strong> Strains expressing a single-copy of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> and/or sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> (CF4582, CF4587, CF4588 and CF4610) will be made available via the <em>Caenorhabditis</em> Genetics Center (CGC). The vector pJG100 carrying <em>Peft-3::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3’UTR</em>, is deposited, along with sequence and map at Addgene. Other strains and plasmids are available upon request. The authors state that all data necessary for confirming the conclusions presented here are represented fully within the article. A detailed protocol to generate <em>C. elegans</em> with sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> and/or wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> integrants is available at <a href="https://doi.org/10.17504/protocols.io.bamkic4w">doi.org/10.17504/protocols.io.bamkic4w</a>. <a href="./appendix.html">Supplemental material</a> will be made available at Figshare.</p> 

<a id="Results" href="#Results" target="_self">
<h3>Results</h3></a>

<h4>Split-wrmScarlet</h4>
<p>To engineer split-wrmScarlet, first we introduced a 32 amino acid spacer between the 10th and 11<sup>th</sup> β-strands of full-length wrmScarlet, following a strategy described previously 

[<a class="citation" href="https://doi.org/10.1038/s41467-017-00494-8" title="Feng, S. et al. &quot;Improved split fluorescent proteins for endogenous protein labeling.&quot; Nature Communications 8, 370 (2017).">Feng 2017</a>]

. We subjected the spacer-inserted wrmScarlet sequence to several rounds of semi-random mutagenesis in <em>E. coli</em>, generating a split version of wrmScarlet with fluorescence comparable to the full-length wrmScarlet when expressed in bacteria. However, upon separating the two fragments into two <em>S. cerevisiae</em> plasmids to test for complementation, we observed no detectable fluorescence in yeast. We decided to continue with several rounds of selection of new mutant libraries in yeast using FACS, by fusing the wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> sequence (without the MDELYK C-terminus residues ) from our brightest <em>E. coli</em> clone to a plasma-membrane targeted blue FP (mTagBP2), and expressing soluble mScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> from a high-copy number vector containing a strong promoter. The brightest resulting protein, which we named split-wrmScarlet, contained 10 amino acid substitutions relative to the C-terminal truncated wrmScarlet (<a href="./appendix.html#Figure_S1">Figure S1, A and B)</a>. Fluorescent microscopy of yeast containing both plasmids corroborated that split-wrmScarlet showed the expected membrane localization and can reach brightness comparable to that of intact wrmScarlet in yeast (<a href="./appendix.html#Figure_S2">Figure S2, A and B</a>). </p>

<h4>Split-wrmScarlet is three-fold brighter than split-sfCherry3 in <em>C. elegans</em> muscles</h4>
<p>In order to compare split-wrmScarlet to split-sfCherry3, the brightest published red split-FP at the time of the experiment, we combined the FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> and FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> fragments into a single plasmid for each fluorophore. Specifically, we generated plasmids encoding three nuclear localization signals (NLS), mTagBFP2, FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>, a T2A peptide bond skipping sequence, mNeonGreen and the corresponding FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>, driven by the ubiquitous somatic <em>Peft-3</em> promoter (<a href="./appendix.html#Figure_S3">Figure S3A</a>). Each FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> was linked to mTagBFP2 in order to reduce the risk of proteolysis of the short peptide, and mNeonGreen was linked to FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> to monitor its expression, and for normalization purposes. Each construct was injected into wild-type animals and fluorescent progeny were analyzed. Unexpectedly, split-sfCherry3 turned out to be toxic when expressed ubiquitously, whereas 99% of split-wrmScarlet-overexpressing worms became viable adults (<a href="./appendix.html#Figure_S3">Figure S3B</a>).</p>
<p>In an attempt to reduce split-sfCherry3-associated toxicity, we modified our construct by using the muscle-specific <em>myo-3</em> promoter and removing the NLS sequence (Figure 1B). We did not detect toxicity associated with the expression of these constructs and were able to compare the fluorescence of split-sfCherry3 and split-wrmScarlet in young adults. Red fluorescence emitted from split-wrmScarlet was 2.9-fold higher than that of split-sfCherry3 when normalized to the mNeonGreen signal (Figure 1, B and C). </p>


<figure id="Figure_1">
<img src="./images/figure_1/figure_1.png"
 alt="Figure 1" id="Figure_1_image">

<figcaption><a href="#Figure_1" target="_self"><strong>Figure 1</strong></a><strong>: Engineering and evaluating split-wrmScarlet.</strong> (A) Principle of endogenous protein labeling with split-wrmScarlet. (B) Schematic of the plasmids encoding split-wrmScarlet and split-sfCherry3. Each plasmid consists of the large FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> sequence fused to mNeonGreen, and the corresponding small FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> sequence fused to the mTagBFP2. The T2A sequence ensures that mTagBFP2::FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> and the corresponding mNeonGreen::FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> are separated. The images are representative displays of the ratio of red to green fluorescence intensity from images acquired under identical conditions after background subtraction and masking with the same threshold. Scale bar, 50 µm. (C) Emission intensities from split-sfCherry3 and split-wrmScarlet normalized to mNeonGreen. Mean ± s.d. Circles are individuals (n=6 for each split protein). ****P &lt; 0.0001.
</figcaption>
</figure><p></p>




<h4>wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>-mediated tagging</h4>
<p>Our protein-tagging approach was analogous to existing split-FP methods developed for human cells 

[<a class="citation" href="https://doi.org/10.1038/ncomms11046" title="Kamiyama, D. et al. &quot;Versatile protein tagging in cells with split fluorescent protein.&quot; Nature Communications 7, 11046–9 ">Kamiyama 2016</a>, 
<a class="citation" href="https://doi.org/10.1073/pnas.1606731113" title="Leonetti, M. D., Sekine, S., Kamiyama, D., Weissman, J. S. &amp; Huang, B. &quot;A scalable strategy for high-throughput GFP tagging of endogenous human proteins.&quot; Natl. Acad. Sci. U.S.A. 113, E3501–8 (2016). ">Leonetti 2016</a>] 
and <em>C. elegans</em> [<a class="citation" href="https://doi.org/10.1534/g3.119.400162" title="Hefel, A. &amp; Smolikove, S. &quot;Tissue-Specific Split sfGFP System for Streamlined Expression of GFP Tagged Proteins in the Caenorhabditis elegans Germline.&quot; G3 9, 1933–1943 (2019). ">Hefel 2019</a>]. 

It requires wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> (i.e. wrmScarlet without the 11<sup>th</sup> β-strand) to be expressed in the cell or tissue of interest, and the short wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> fragment to be inserted at an endogenous locus to tag a protein of interest (Figure 1A).
To build strains expressing single-copy insertions of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>, we first optimized its sequence for <em>C. elegans</em> codon usage 

[<a class="citation" href="https://doi.org/10.1038/nmeth.1565" title="Redemann, S. et al. &quot;Codon adaptation-based control of protein expression in C. elegans.&quot; Nature Publishing Group 8, 250–252 (2011).">Redeman 2011</a>]

 and included three introns (<a href="./appendix.html#Tables">Supplementary Tables</a>). The strain expressing wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in all somatic cells (driven by the <em>eft-3</em> promoter and <em>unc-54</em> 3’UTR) was generated by editing the genome of the existing MosSCI line CA1200 
 
 [<a class="citation" href="https://doi.org/10.1242/dev.129635" title="Zhang, L., Ward, J. D., Cheng, Z. &amp; Dernburg, A. F. &quot;The auxin-inducible degradation (AID) system enables versatile conditional protein depletion in C.elegans.&quot; Development 142, 4374–4384 (2015).">Zhang 2015</a>]
 
  and replacing the sequence encoding tir-1::mRuby with wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> using CRISPR/Cas9 and hybrid DNA templates 
  
  [<a class="citation" href="https://doi.org/10.1534/genetics.115.179382" title="Paix, A. et al. &quot;High Efficiency, Homology-Directed Genome Editing in Caenorhabditis elegans Using CRISPR-Cas9 Ribonucleoprotein Complexes.&quot; Genetics 201, 47-54 (2015).">Paix 2015</a>,
   <a class="citation" href="https://doi.org/10.1534/genetics.118.301532" title="Dokshin, G. A., Ghanta, K. S., Piscopo, K. M. &amp; Mello, C. C. &quot;Robust Genome Editing with Short Single-Stranded and Long, Partially Single-Stranded DNA Donors in Caenorhabditis elegans.&quot; Genetics 210, 781–787 (2018).">Dokshin 2018</a>]
  
   (<a href="./appendix.html#Tables">Supplementary Tables</a>). In order to perform tissue-specific labeling, we generated a strain expressing muscle-specific wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> using the SKI-LODGE system in the strain WBM1126 
   
   [<a class="citation" href="https://doi.org/10.1534/g3.119.400314" title="Silva-García, C. G. et al. &quot;Single-Copy Knock-In Loci for Defined Gene Expression in Caenorhabditis elegans.&quot; G3 9, 2195–2198 (2019).">Silva-García 2019</a>]
   
    (<a href="./appendix.html#Tables">Supplementary Tables</a>). The expression of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in these two lines did not affect the number of viable progeny (<a href="./appendix.html#Figure_S4">Figure S4A</a>) nor lifespan (<a href="./appendix.html#Figure_S4">Figure S4B</a>), suggesting that the expression of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> has no deleterious effect. To tag a gene of interest with the wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> fragment, we used microinjection of preassembled Cas9 RNPs because this method enables high-efficiency genome editing in worms 
    
    [<a class="citation" href="https://doi.org/10.1534/genetics.115.179382" title="Paix, A. et al. &quot;High Efficiency, Homology-Directed Genome Editing in Caenorhabditis elegans Using CRISPR-Cas9 Ribonucleoprotein Complexes.&quot; Genetics 201, 47-54 (2015).">Paix 2015</a>]. 
    
    The most efficient insertion of short sequences in <em>C. elegans</em> was previously shown to be achieved using ssODN donors 
    
    [<a class="citation" href="https://doi.org/10.1534/genetics.115.179382" title="Paix, A. et al. &quot;High Efficiency, Homology-Directed Genome Editing in Caenorhabditis elegans Using CRISPR-Cas9 Ribonucleoprotein Complexes.&quot; Genetics 201, 47-54 (2015).">Paix 2015</a>, 
    <a class="citation" href="https://doi.org/10.1534/g3.117.300216" title="Prior, H., Jawad, A. K., MacConnachie, L. &amp; Beg, A. A. &quot;Highly Efficient, Rapid and Co-CRISPR-Independent Genome Editing in Caenorhabditis elegans.&quot; G3 7, 3693–3698 (2017).">Prior 2017</a>, 
    <a class="citation" href="https://doi.org/10.1534/genetics.118.301532" title="Dokshin, G. A., Ghanta, K. S., Piscopo, K. M. &amp; Mello, C. C. &quot;Robust Genome Editing with Short Single-Stranded and Long, Partially Single-Stranded DNA Donors in Caenorhabditis elegans.&quot; Genetics 210, 781–787 (2018).">Dokshin 2018</a>]. 
    
    A great advantage of this strategy is that all of the components required for editing are commercially available or can be synthesized rapidly in the lab 
    
    [<a class="citation" href="https://doi.org/10.1073/pnas.1606731113" title="Leonetti, M. D., Sekine, S., Kamiyama, D., Weissman, J. S. &amp; Huang, B. &quot;A scalable strategy for high-throughput GFP tagging of endogenous human proteins.&quot; Natl. Acad. Sci. U.S.A. 113, E3501–8 (2016). ">Leonetti 2016</a>]. 
    
    Synthetic ssODNs have a typical size limit of 200 nt. The small size of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> (18 a.a.) is key: 200 nt can encompass wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> (66 nt, including a 4 a.a. linker) flanked by two 67 nt homology arms for HDR. In principle, a few days after the somatic and/or muscle-specific wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> strain(s) are microinjected, progeny can be screened for red fluorescence, genotyped and sequenced to check the accuracy of editing (Figure 2; a detailed protocol is available <a href="https://doi.org/10.17504/protocols.io.bamkic4w">doi.org/10.17504/protocols.io.bamkic4w</a>). If desired, co-CRISPR strategies such as <em>dpy-10(cn64)</em> 
    
    [<a class="citation" href="https://doi.org/10.1534/genetics.115.179382" title="Paix, A. et al. &quot;High Efficiency, Homology-Directed Genome Editing in Caenorhabditis elegans Using CRISPR-Cas9 Ribonucleoprotein Complexes.&quot; Genetics 201, 47-54 (2015).">Paix 2015</a>] 
    or co-injection with pRF4 
[<a class="citation" href="https://doi.org/10.1534/genetics.118.301532" title="Dokshin, G. A., Ghanta, K. S., Piscopo, K. M. &amp; Mello, C. C. &quot;Robust Genome Editing with Short Single-Stranded and Long, Partially Single-Stranded DNA Donors in Caenorhabditis elegans.&quot; Genetics 210, 781–787 (2018).">Dokshin 2018</a>]

 can be used to help screening for correct candidates and to control for microinjection efficacy and payload toxicity.</p>

<figure id="Figure_2">
<img src="./images/figure_2/figure_2.png"
 alt="Figure 2" id="Figure_2_image">

<figcaption><a href="#Figure_2" target="_self"><strong>Figure 2</strong></a><strong>: wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>-mediated tagging.</strong> Schematic representation of the split-wrmScarlet workflow to visualize endogenous proteins in muscles or somatic tissues. Some illustrations were created with BioRender.com.
</figcaption>
</figure><p></p>



<p>To test our approach, we used it to tag six proteins with distinct subcellular localizations. Starting with the somatic wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> parental strain CF4582, we introduced wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> at the N-terminus of TBB-2, FIB-1 or VHA-13 or at the C-terminus of EAT-6, HIS-3 and TOMM-20 (<a href="./appendix.html#Tables">Supplementary Tables</a>). These proteins mark the cytoskeleton, nucleoli, lysosomes, plasma membrane, nuclei and mitochondria, respectively. Importantly, for transmembrane targets, the wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> tag was introduced at the terminus exposed to the cytosol. wrmScarlet fluorescence from all six proteins matched their expected subcellular localization in somatic cells (Figure 3, A-F). To test the muscle-specific wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> line CF4610, we tagged the N-terminus of the endogenous FIB-1 with wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> and confirmed the fluorescence from nucleoli in muscle cells (Figure 3, G and H). Together, our results show that split-wrmScarlet enables rapid fluorescent tagging of proteins with disparate cytoplasmic or nuclear locations expressed from their endogenous loci.</p>

<figure id="Figure_3">
<img src="./images/figure_3/figure_3.png"
 alt="Figure 3" id="Figure_3_image">

<figcaption><a href="#Figure_3" target="_self"><strong>Figure 3</strong></a><strong>: Split-wrmScarlet labeling of proteins with distinct subcellular locations.</strong> Endogenous proteins tagged with wrmScarlet11 in animals expressing wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in somatic tissues or in muscles. (A-F) Confocal images of worms expressing somatic wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> and (A) EAT-6::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> (membrane), (B) wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::TBB-2 (cytoskeleton), (C) wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::FIB-1 (nucleoli), (D) HIS-3::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> (nuclei), (E) wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::VHA-13 (lysosomes), or (F) TOMM-20::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> (mitochondria). (G,H) Transgenic worms expressing wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in muscle and wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::FIB-1 (A-G) Maximum intensity projections of 3D stacks shown. Scale bars, 50 µm.
</figcaption>
</figure><p></p>

<h4>wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> tandem repeats increase fluorescence</h4>
<p>To benchmark the fluorescence intensity of split-wrmScarlet to its full-length counterpart, we first generated wrmScarlet::vha-13 transgenic animals and compared their fluorescence to wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::vha-13 in worms expressing wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> somatically (Figure 4, A and B). At the vha-13 locus, split-wrmScarlet was about half as bright as a full-length fluorophore (48%), a ratio comparable to that of split-mNeonGreen2 and its full-length counterpart in human cells 

[<a class="citation" href="https://doi.org/10.1038/s41467-017-00494-8" title="Feng, S. et al. &quot;Improved split fluorescent proteins for endogenous protein labeling.&quot; Nature Communications 8, 370 (2017).">Feng 2017</a>].</p>

<figure id="Figure_4">
<img src="./images/figure_4/figure_4.png"
 alt="Figure 4" id="Figure_4_image">

<figcaption><a href="#Figure_4" target="_self"><strong>Figure 4</strong></a><strong>: wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> tandem repeats increase fluorescence.</strong> (A) Images of animals carrying either full-length wrmScarlet, wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> or two tandem repeats of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> inserted at the endogenous VHA-13 N-terminus. (B) wrmScarlet emission intensities of animals carrying full-length wrmScarlet, wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> or dual wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> inserted at the VHA-13 N-terminus. Mean ± s.d. Circles are individuals. ***P &lt; 0.001. (C) Images of animals carrying either a single wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> or three tandem repeats of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> inserted at the HIS-3 C-terminus. (D) wrmScarlet emission intensities from animals carrying a single wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> or three tandem repeats of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> knockin at the HIS-3 C-terminus. Mean ± s.d. Circles are individuals. ****P &lt; 0.0001, ***P &lt; 0.001, **P &lt; 0.005. Images from each comparison were taken under identical instrument conditions using confocal microscopy and are shown using identical brightness and contrast settings. Images shown are from a single confocal plane. Scale bars, 50 µm.
</figcaption>
</figure><p></p>

<p>Since visualizing endogenous proteins of low abundance can be challenging, it is key to address this limitation. Increasing the number of FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> domains tagged to an endogenous protein multiplies the number of the corresponding FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> recruited, increasing the overall fluorescent signal in human cells 

[<a class="citation" href="https://doi.org/10.1073/pnas.1606731113" title="Leonetti, M. D., Sekine, S., Kamiyama, D., Weissman, J. S. &amp; Huang, B. &quot;A scalable strategy for high-throughput GFP tagging of endogenous human proteins.&quot; Natl. Acad. Sci. U.S.A. 113, E3501–8 (2016). ">Leonetti 2016</a>] 
and in <em>C. elegans</em> [<a class="citation" href="https://doi.org/10.1534/genetics.119.302063" title="He, S., Cuentas-Condori, A. &amp; Miller, D. M. &quot;NATF (Native and Tissue-Specific Fluorescence): A Strategy for Bright, Tissue-Specific GFP Labeling of Native Proteins in Caenorhabditis elegans.&quot; Genetics 212, 387–395 (2019).">He 2019</a>, 
<a class="citation" href="https://doi.org/10.1534/g3.119.400162" title="Hefel, A. &amp; Smolikove, S. &quot;Tissue-Specific Split sfGFP System for Streamlined Expression of GFP Tagged Proteins in the Caenorhabditis elegans Germline.&quot; G3 9, 1933–1943 (2019). ">Hefel 2019</a>]. 

To test whether split-wrmScarlet fluorescence would be enhanced by wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> tandem repeats, we introduced two wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> domains at the N-terminus of vha-13 and three wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> domains at the C-terminus of his-3, using CRISPR/Cas9 and dsDNA as donor template (<a href="./appendix.html#Tables">Supplementary Tables</a>), in animals expressing somatic wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>. Compared to animals carrying a single wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> at the identical locus, carrying a tandem of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> increased overall fluorescence by 1.5-fold, while carrying three increased it by 2.3-fold (Figure 4, C and D). Therefore, increasing the number of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> repeats improves visualization of low-abundance proteins.</p>

<h4>sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>-mediated tagging in somatic cells</h4>
<p>Split-sfGFP has been used successfully in worms before 

[<a class="citation" href="https://doi.org/10.7554/eLife.26376" title="Noma, K., Goncharov, A., Ellisman, M. H. &amp; Jin, Y.&quot;Microtubule-dependent ribosome localization in C. elegans neurons.&quot; eLife 6, 218 (2017).">Noma 2017</a>,
<a class="citation" href="https://doi.org/10.7554/eLife.26376" title="He, S., Cuentas-Condori, A. &amp; Miller, D. M. &quot;NATF (Native and Tissue-Specific Fluorescence): A Strategy for Bright, Tissue-Specific GFP Labeling of Native Proteins in Caenorhabditis elegans.&quot; Genetics 212, 387–395 (2019).">He 2019, </a> <a class="citation" href="https://doi.org/10.1534/g3.119.400162" title="Hefel, A. &amp; Smolikove, S. &quot;Tissue-Specific Split sfGFP System for Streamlined Expression of GFP Tagged Proteins in the Caenorhabditis elegans Germline.&quot; G3 9, 1933–1943 (2019). ">Hefel 2019</a>]. 

However, there was still a need for a strain that ubiquitously expressed sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in the soma from an integrated single-copy insertion in order to avoid heterogeneity of expression, and time-consuming manual maintenance. To build this strain, we first optimized the original sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> sequence for <em>C. elegans</em> codon usage and included one intron 

[<a class="citation" href="https://doi.org/10.1038/nbt1044" title="Cabantous, S., Terwilliger, T. C. &amp; Waldo, G. S. &quot;Protein tagging and detection with engineered self-assembling fragments of green fluorescent protein.&quot; Nat Biotechnol 23, 102–107 (2004).">Cabantous 2004</a>, 
<a class="citation" href="https://doi.org/10.1038/nmeth.1565" title="Redemann, S. et al. &quot;Codon adaptation-based control of protein expression in C. elegans.&quot; Nature Publishing Group 8, 250–252 (2011).">Redeman 2011</a>] 

(<a href="./appendix.html#Tables">Supplementary Tables</a>). We initially generated a strain expressing sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> driven by the <em>let-858</em> promoter and <em>unc-54</em> 3’UTR using MosSCI (<a href="./appendix.html#Tables">Supplementary Tables</a>), but later replaced the <em>let-858</em> promoter with the <em>eft-3</em> promoter using CRISPR/Cas9 and hybrid DNA donor template because we observed that Peft-3 resulted in significantly higher levels of gene expression 

[<a class="citation" href="https://doi.org/10.1534/genetics.115.179382" title="Paix, A. et al. &quot;High Efficiency, Homology-Directed Genome Editing in Caenorhabditis elegans Using CRISPR-Cas9 Ribonucleoprotein Complexes.&quot; Genetics 201, 47-54 (2015).">Paix 2015</a>, 
<a class="citation" href="https://doi.org/10.1534/genetics.118.301532" title="Dokshin, G. A., Ghanta, K. S., Piscopo, K. M. &amp; Mello, C. C. &quot;Robust Genome Editing with Short Single-Stranded and Long, Partially Single-Stranded DNA Donors in Caenorhabditis elegans.&quot; Genetics 210, 781–787 (2018).">Dokshin 2018</a>]

 (<a href="./appendix.html#Tables">Supplementary Tables</a>). To validate this strain, we inserted sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> at the N-terminus of lysosomal VHA-13 or at the C-terminus of nuclear-localized HIS-3 (Figure 5, A and B). Both strains yielded relatively bright signals in accordance with their predicted subcellular localization.</p>

<figure id="Figure_5">
<img src="./images/figure_5/figure_5.png"
 alt="Figure 5" id="Figure_5_image">

<figcaption><a href="#Figure_5" target="_self"><strong>Figure 5</strong></a><strong>: Split-sfGFP and split-wrmScarlet dual color protein labeling.</strong> Images of animals stably expressing sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in somatic tissues (A) CF4592 (<em>muIs253[Peft-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3'UTR, Cbr-unc-119(+)] II; unc-119(ed3) III; his-3(muIs255[his-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>] V</em>) or (B) CF4589 (<em>muIs253[Peft-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>::unc-54 3'UTR, Cbr-unc-119(+)] II; unc-119(ed3) III; vha-13(muIs268[sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::vha-13]) V</em>
). (C) Dual color protein labeling with split-wrmScarlet and split-sfGFP in somatic cells. Composite display of red and green channels of animals expressing wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> and sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in somatic tissues, HIS-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> and wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::FIB-1; CF4602 (<em>muIs253[Peft-3::sfGFP1-10::unc-54 3'UTR, Cbr-unc-119(+)], muIs252[Peft-3::wrmScarlet1-10::unc-54 3'UTR, Cbr-unc-119(+)] II; unc-119(ed3) III; fib-1(muIs254[wrmScarlet11::fib-1]), his-3(muIs255[his-3::sfGFP11]) V</em>). Maximum intensity projections of 3D stacks shown. Scale bars, 50 µm.
</figcaption>
</figure><p></p>



<h4>Dual color protein labeling with split-wrmScarlet and split-sfGFP </h4>
<p>Finally, to test the compatibility of split-wrmScarlet and split-sfGFP in vivo, we crossed the strains <em>Peft-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>; his-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub></em> (CF4592) and <em>Peft-3::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>; wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::fib-1</em> (CF4601). This cross resulted in the generation of the line <em>Peft-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>, Peft-3::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub></em> (CF4588) as well as the dually labeled strain <em>Peft-3::sfGFP1-10, Peft-3::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>; wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>::fib-1, his-3::sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub></em> (CF4602, Figure 5C). The fluorescent signals from both split-FPs appeared in their respective subcellular compartments, strongly suggesting the absence of interference between the two systems. We would like to note an additional advantage of the strain CF4588. The loci of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> and sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> are genetically linked (only 0.96 cM apart), which facilitates outcrossing when needed. In addition, our <em>C. elegans</em> lines expressing wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> and sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> are viable homozygotes, so the strains do not require special maintenance.</p>

<h4>Split-wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> was not functional in the <em>C. elegans</em> germline or mammalian cells</h4>
<p>In an attempt to generate a strain with tissue-specific expression of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> in the germline, we optimized the sequence of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> with three introns and engineered it to avoid piRNA recognition in order to prevent transgene silencing 

[<a class="citation" href="https://doi.org/10.1093/nar/gky277" title="Wu, W.-S. et al. &quot;Efficient Generation of pirScan: a webserver to predict piRNA targeting sites and to avoid transgene silencing in C. elegans.&quot; Nucleic Acids Res 46, W43–W48 (2018).">Wu 2018</a>, 
<a class="citation" href=" https://doi.org/10.1126/science.aao2840" title="Zhang, D. et al. &quot;The piRNA targeting rules and the resistance to piRNA silencing in endogenous genes.&quot;  Science 359, 587–592 (2018). ">Zhang 2018</a>] 

(<a href="./appendix.html#Tables">Supplementary Tables</a>). For reasons we do not understand, we were unable to detect wrmScarlet fluorescence in the germline wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> strain we generated, when injecting a plasmid encoding mNeonGreen::linker::wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> (<a href="./appendix.html#Figure_S5">Figure S5, A and B</a>), despite detecting mNeonGreen fluorescence, potentially due to compromised expression, folding or maturation of the protein wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>. A similar negative result was obtained when attempting to express split-mScarlet in mammalian cells, in spite of efforts to rescue its fluorescence by screening an mScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> single/double mutant library in HEK293T cells (<a href="./appendix.html#Figure_S6">Figure S6</a>, A and B, and <a href="./appendix.html#supplementary_text">supplementary text</a>). </p>

<a id="Discussion" href="#Discussion" target="_self">
<h3>Discussion</h3></a>
<p>Several considerations should be taken into account when using this method. First, as with all existing split-FP systems, detection of a given protein labeled with a FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> can only occur in a cellular compartment where the corresponding FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> is present. Proteins tagged with wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> or sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> generated in this work were either exposed to the cytosol or nucleoplasm (nuclei or nucleoli), where both wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> and/or sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> were present. For proteins or epitopes located within the lumen of organelles, such as the endoplasmic reticulum or mitochondria, one might need to generate and validate <em>C. elegans</em> lines expressing wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> or sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> containing a mitochondrial localization sequence or ER signal peptide and retention signals, respectively. These approaches have been used successfully in mammalian cells with split-sfGFP when tagging ER-resident polypeptides 

[<a class="citation" href="https://doi.org/10.1038/ncomms11046" title="Kamiyama, D. et al. &quot;Versatile protein tagging in cells with split fluorescent protein.&quot; Nature Communications 7, 11046–9 ">Kamiyama 2016</a>] and with split-sfCherry2 to detect protein present in the mitochondrial matrix 

[<a class="citation" href="https://doi.org/10.3390/cells8121583" title="Ramadani-Muja, J. et al. &quot;Visualization of Sirtuin 4 Distribution between Mitochondria and the Nucleus, Based on Bimolecular Fluorescence Self-Complementation.&quot; Cells 8(12), 1583 (2019). ">Ramadani-Muja 2019</a>].</p>


<p>Second, as for any other protein tag, it is important to select, when possible, a site that is unlikely to interfere with protein folding, function or localization 

[<a class="citation" href="https://doi.org/10.1002/0471143030.cb2104s27" title="Snapp, E. &quot;Design and use of fluorescent fusion proteins in cell biology.&quot; Current Protocols in Cell Biology Chapter 21, 21.4.1–21.4.13 (2005).">Snapp 2005</a>, 
<a class="citation" href="https://doi.org/10.1534/genetics.119.301506" title="Nance, J. &amp; Frøkjær-Jensen, C. &quot;The Caenorhabditis elegans Transgenic Toolbox.&quot; The EMBO Journal 10, 3959–3970 (1991).">Nance 2019</a>]. 

For example, N-termini of membrane- and organelle-resident proteins often contain signal peptides or localization signals, and C-termini may contain sequences that regulate protein turnover (degrons). Interestingly, there are examples of proteins that become toxic when tagged with a full-length GFP, but tolerate labeling with a split protein. For example, SYP-4 was reported to be mostly functional when endogenously-tagged with sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> in a strain expressing sfGFP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> specifically in the germline, but not functional when labeled with full-length GFP 

[<a class="citation" href="https://doi.org/10.1534/g3.119.400162" title="Hefel, A. &amp; Smolikove, S. &quot;Tissue-Specific Split sfGFP System for Streamlined Expression of GFP Tagged Proteins in the Caenorhabditis elegans Germline.&quot; G3 9, 1933–1943 (2019). ">Hefel 2019</a>]. 

For proteins of interest present at low levels, we provided an alternative protocol to insert an additional two or three wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> fragments, which increases the overall fluorescence substantially. However, the number of wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> fragments could likely be increased further, to at least seven tandem repeats, based on approaches used successfully with split-sfGFP in human cells 

[<a class="citation" href="https://doi.org/10.1038/s41467-017-00494-8" title="Feng, S. et al. &quot;Improved split fluorescent proteins for endogenous protein labeling.&quot; Nature Communications 8, 370 (2017).">Feng 2017</a>] and <em>C. elegans</em> 

[<a class="citation" href="https://doi.org/10.7554/eLife.26376" title="Noma, K., Goncharov, A., Ellisman, M. H. &amp; Jin, Y. &quot;Microtubule-dependent ribosome localization in C. elegans neurons.&quot; eLife 6, 218 (2017).">Noma 2017</a>, 


<a class="citation" href="https://doi.org/10.7554/eLife.26376" title="He, S., Cuentas-Condori, A. &amp; Miller, D. M. &quot;NATF (Native and Tissue-Specific Fluorescence): A Strategy for Bright, Tissue-Specific GFP Labeling of Native Proteins in Caenorhabditis elegans.&quot; Genetics 212, 387–395 (2019).">He 2019</a>,

 <a class="citation" href="https://doi.org/10.1534/g3.119.400162" title="Hefel, A. &amp; Smolikove, S. &quot;Tissue-Specific Split sfGFP System for Streamlined Expression of GFP Tagged Proteins in the Caenorhabditis elegans Germline.&quot; G3 9, 1933–1943 (2019). ">Hefel 2019</a>].</p>


<p>Third, we would like to emphasize differences between our technique and the bimolecular fluorescence-complementation (BiFC) assay. When used together, the green and red split fluorescent proteins used here can provide information on co-localization, but unlike BiFC split proteins 

[<a class="citation" href="https://doi.org/10.1016/S1097-2765(02)00496-3" title="Hu, C. D., Chinenov, Y. &amp; Kerppola TK. &quot;Visualization of interactions among bZIP and Rel family proteins in living cells using bimolecular fluorescence complementation.&quot; Molecular Cell 9, 789-98 (2002). ">Hu 2002</a>], 

they are not intended to assess protein-protein interactions directly. This is because BiFC split proteins require finely tuned weak affinities that do not disrupt the underlying interaction being studied. In our approach, only the wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> fragment is attached to a protein of interest, the wrmScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> one is expressed in excess and untagged. In principle, it could be possible to use secondary assays that assess target protein interactions using split-FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub>/FP<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub> proteins, like FRET or anisotropy, if one so desired and one can account for the possibility of incomplete complementation. </p>
<p>Lastly, we would like to note that in spite of its being three times brighter than the latest split-sfCherry3 in worms, our current split-wrmScarlet was not visible in the mammalian cell line we examined (<a href="./appendix.html#Figure_S6">Figure S6</a>). Its ability to fluoresce is not restricted to worms, because it can reach wild-type levels of brightness in yeast. We do not know the basis for this discrepancy, nor why the protein is not visible in the <em>C. elegans</em> germline. It is possible that the concentration of the mScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">1-10</sub> fragment in mammalian cells is too low to drive complementation with mScarlet<sub style="font-size: 10px; line-height: 0; vertical-align: -5px;">11</sub>. This could potentially be overcome by further mutagenizing split-mScarlet and screening for fluorescence at low expression levels in mammalian cells. </p>
<p>In conclusion, we believe our system can substantially increase the speed, efficiency, and easiness of in vivo microscopy studies in <em>C. elegans</em>. We expect it to facilitate two-color and co-localization experiments and to find wide use in the worm community. We believe that these strains could facilitate novel or large-scale experiments, such as efforts to tag the entire genome of <em>C. elegans</em>. </p>

<h3>Author contributions</h3>
<p>M.I. developed the split-wrmScarlet in A.G.Y. laboratory. J.P. performed the cell sorting. J.G. performed <em>C. elegans</em> experiments in C.K. laboratory. L.S. conducted the mammalian cell experiments in M.D.L. laboratory. J.G. wrote the initial draft. All authors provided intellectual contributions to the collaboration. </p>

<a id="Acknowledgements" href="#Acknowledgements" target="_self"></a>
<h3>Acknowledgements</h3>
<p>We thank Katie Podshivalova, Rex Kerr, Calvin Jan and David Botstein for comments on the manuscript, Peichuan Zhang for experimental suggestions, and members of the Kenyon lab for fruitful discussions. We also thank Behnom Farboud from Barbara Meyer's laboratory for discussing CRISPR/Cas9 protocols and Liangyu Zhang from Abby Dernburg's laboratory for sharing sequences of the CA1200 strain. Some strains were provided by the CGC, which is funded by NIH Office of Research Infrastructure Programs (P40 OD010440). This project was supported by Calico Life Sciences LLC.</p>

<h3><a href="./appendix.html">Appendix</a></h3>
<p>Additional details and discussion can be found in the <a href="./appendix.html">appendix</a>, which is also referenced via hyperlinks throughout this article.</p>




</section>
<footer>
  <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="./images/cc_by_4p0.png"></a>
  
  <p><small>Hosted on <a href="https://pages.github.com">GitHub Pages</a></small></p>
</footer>
</div>
 <!--[if !IE]><script>fixScale(document);</script><![endif]-->
</body>
</html>
